docs/Building-Transmission.md

```diff
@@ -120,24 +120,35 @@ $ sudo make install
 ```
 
 ## On Windows ##
-For Windows XP and above there are several choices:
 
-### Cygwin environment ###
-With Cygwin https://cygwin.com/ installed, the CLI tools (transmission-remote, transmission-cli, etc.) and the daemon can be built easily.
+## Building transmission-daemon
+You need the following installed:
 
-No patches needed(\*), all the recent versions of Transmission build almost out-of-the-box (you need to install the prerequisites), and the CLI tools work better under Cygwin than those built with MinGW.
+* Visual Studio 2019 or greater(the Community Edition is sufficient - just make sure its C++ compiler, MSVC, is installed)
```

Comment:
nit: can we have a space between "greater" and "("?

Reply:
Yeah, i corrected it